py/client/pydeephaven/session.py

```diff
@@ -413,12 +400,44 @@ def open_table(self, name: str) -> Table:
             except Exception as e:
                 if isinstance(e.__cause__, grpc.RpcError):
                     if e.__cause__.code() == grpc.StatusCode.INVALID_ARGUMENT:
-                        raise DHError(f"no table by the name {name}") from None
+                        raise DHError(not_found_err) from None
                 raise e
             finally:
                 # Explicitly close the table without releasing it (because it isn't ours)
                 faketable.ticket = None
                 faketable.schema = None
+        
+    def ticket_to_table(self, ticket: str) -> Table:
+        """Opens a table in the global scope for the given ticket on the server.
```

Comment:
Does it have to be in the global scope?  I have no idea how tickets work.

Reply:
I think so, I am not an expert on the arrow/ticket stuff, but given how `open_table` was implemented based on the same code I think that statement has to be true.

Reply:
This is nothing to do with global scope, it just lets you take a string (should be bytes?). The existing method `open_table` is the "open a table in the global scope", and is documented as such. It looks like the comment was copied directly... and from the tests, it looks like the use case actually wants to call `open_table`?

Reply:
I am not crazy with the name `ticket_to_table`, would `open_ticket` be better and more consistent with the naming of other methods?

Reply:
Also @niloc132 found that in _console_services, we have an `open_table` that is so out of date it is wrong and not used anymore, @jcferretti Could you remove it?